src/Gui/SoFCColorBar.cpp

```diff
@@ -123,15 +123,15 @@ float SoFCColorBarBase::getBounds(const SbVec2s& size, float& fMinX, float&fMinY
     // The cam height is set in SoFCColorBarBase::getBoundingWidth to 10.
     // Therefore the normalized coordinates are in the range [-5, +5] x [-5ratio, +5ratio] if ratio > 1
     //  and [-5ratio, +5ratio] x [-5, +5] if ratio < 1.
-    // We don't want the whole height covered by the color bar (to have e.g space to the axis cross)
-    // thus we take as base 4.
-    float baseYValue = 4.0f;
+    // We don't want the whole height covered by the color bar (to have e.g space to the axis cross
+    // and the Navigation Cube) thus we take as base 3.25.
+    float baseYValue = 3.25f;
```

Comment:
Works well on "standard case" -- Full HD screen, FC maximized -- but overlay with NaviCube as soon as viewport is reduced (for ex. open the ReportView or minimize FC window).
Wonder how it looks on HiDPI screens.